src/components/app.js

```diff
@@ -227,7 +227,7 @@ const App = () => {
                 <React.Fragment>
                     <CssBaseline />
                     <TopBar
-                        appName="Actions"
+                        appName="Data"
                         appColor="#DA0063"
                         appLogo={
                             theme === LIGHT_THEME ? (
```

Comment:
Rename GridActionsLogoDark and GridActionsLogoLight component names
And the associated svg files : GridActions_logo_dark.svg and GridActions_logo_light.svg